CHANGELOG.md

```diff
@@ -1,43 +1,116 @@
-* Added a parser for `Obituary_Deh_Actor` DEHACKED strings (thanks to @SirBofu).
-* Save files now use UMAPINFO labels instead of map markers.
-* Moved secret message higher, and prevented it from disabling the crosshair.
-* Added `snd_limiter` config key to toggle the sound limiter.
-* Restored "Resampler" option in the "General: Audio" menu.
-* The order in which automap lines are drawn has been changed to improve visual consistency.
+
+* Mod Support:
+  - Added support for ID24 `SKYDEFS` and intermissions.
+  - Added support for ID24 `SBARDEF` with additional widescreen layout and widgets. Remove Boom HUD.
+  - Added support for PNG graphics, textures, flats and sprites. Automatically convert PNG palette. Convert 32-bit PNG to 8-bit with color degradation (not recommended).
+  - Added resource directories and partial support for PK3 archives.
+  - Added VX_START/VX_END namespace for voxels.
+  - Added support for `TRAKINFO`. Remixed soundtrack from "Doom + Doom II". Autoload extras.wad if found.
+
+* Video:
+  - Added 32:9 aspect ratio support for ultrawide monitors.
+  - Draw sprites overlapping into visible sectors. Enhancement to stop things from disappearing due to "anchor" sector not being in line of sight (thanks to @SirBofu)
+  - Limit max distance for sprites to 8192 map units as before (improve performance in comatose.wad).
+  - Improve FPS limiter (thanks to @gendlin).
+
+* Sound:
+  - Added 4-band equalizer for sound effects.
+  - Added output limiter and SFX channels options to menu.
+  - Added OpenAL Doppler effect option to menu.
+  - Added click/pop prevention for sound lumps that start or end with a non-zero amplitude.
+
+* Music:
+  - Added auto gain (loudness normalization) menu option for music. Applies to sampled music (mp3/ogg/xm/etc.), FluidSynth, and OPL3.
+  - Added support for multiple OPL3 chips, various improvements (thanks to @jpernst)
+  - Changed native MIDI music volume curve to linear.
+  - Added FluidSynth and OPL3 menu items.
+  - Updated default FluidSynth reverb and chorus parameters. These parameters are configurable in woof.cfg.
+  - Improved EMIDI support to match behavior of Build engine games.
+  - Added DMXOPL enhanced FM patch set.
+
+* Input:
+  - Added option for fake high-resolution turning when using short tics. See `fake_longtics` in woof.cfg.
+  - Added quickstart cache option (from DSDA-Doom). See `quickstart_cache_tics` in woof.cfg.
+  - Added "Last" weapon key binding.
+  - Added "Cycle Through All Weapons" option.
+  - Added support for SSG key binding when using vanilla compatibility level.
+  - Improved peformance when handling many input events at once.
+  - Optimized input polling for older computers (thanks to @gendlin).
+
+* [Gamepad](https://github.com/fabiangreffrath/woof/wiki/Gamepad):
+  - Added custom weapon slots. The default configuration uses the d-pad and matches the Doom II Xbox 360 port.
+  - Added rumble support.
```

Comment:
@ceski-1 Could you mention support for custom weapons/sfx? Also, how about turning on the rumble by default?

Reply:
https://github.com/fabiangreffrath/woof/commit/fcdec943c13a3865cfb8aa8d86ea7d91a1da4fa6 and https://github.com/fabiangreffrath/woof/pull/2026/commits/85a6fb761f5d59ba9e028817301a565426c8209f